packages/core/strapi/lib/services/metrics/sender.js

```diff
@@ -44,13 +44,13 @@ module.exports = (strapi) => {
   const anonymousMetadata = {
     environment: strapi.config.environment,
     os: os.type(),
-    osPlatform: os.platform(),
+    platform: os.platform(),
+    osArch: os.arch(),
     osRelease: os.release(),
-    nodeVersion: process.version,
+    nodeVersion: process.versions.node,
```

Comment:
what's the difference with process.version ?

Reply:
I think it is what he mentioned in the PR description:

> Removed the v prefix from the sent nodeVersion.

Reply:
Exactly that, I checked if it would break anything on Amplitude's side but it won't so I just wanted to keep things consistent between versions sent to Amplitude